docs/nodes/mamaki.md

```diff
@@ -55,12 +66,14 @@ cp $HOME/networks/mamaki/genesis.json $HOME/.celestia-app/config
 Set seeds and peers:
 
 ```sh
-SEEDS="74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656"
-PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+SEEDS="f0c58d904dec824605ac36114db28f1bf84f6ea3@144.76.112.238:26656"
```

Comment:
we will probably update our seed node peer since we had to split both rpc and seed node

Reply:
quick command we can use to update bootstrap peers now, instead of seeds, it will take the ones directly from the file on github (caveat: need to approve my PR to rename the file there):

BOOTSTRAP_PEERS=`curl -sL https://raw.githubusercontent.com/celestiaorg/networks/master/mamaki/bootstrap-peers.txt | tr -d '\n'` && echo $BOOTSTRAP_PEERS
sed -i.bak -e "s/^bootstrap-peers *=.*/bootstrap-peers = \"$BOOTSTRAP_PEERS\"/" $HOME/.celestia-app/config/config.toml